MEZFLIPR/iocBoot/iocMEZFLIPR-IOC-01/config.xml

```diff
@@ -4,7 +4,7 @@
 <ioc_desc>The IOC for a Mezei spin flipper</ioc_desc>
 <ioc_details>The IOC for a Mezei spin flipper</ioc_details>
 <macros>
-<macro name="HOST" pattern=".*" description="IP Address or computer name to connect to" hasDefault="NO" />
```

Comment:
They currently use a computer name rather than an IP - can we make this regex generic enough to also accept computer name please?